### PR TITLE
Add QR scan button next to search input on main screen

### PR DESCRIPTION
**Pull request by adzialocha**
2024-02-08-120959_636x119_scrot

Closes: #3622

### CODE REVIEW

File: images/icons/qr.svg
Comment (Simon-Laux) [inline images/icons/qr.svg:1]: we can probably update the references to `../images/qr_icon.svg` and remove the old file so it is not duplicated.
Or just remove the old file and remove the qr button from the sidebar.

File: src/renderer/components/SearchInput/SearchInputButton.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+type Props = {
+  icon: IconName
+  onClick: () => void
+  size?: number
+}
+
+export default function SearchInputButton({ icon, onClick, size = 20 }: Props) {
+  return (
+    <button className={styles.searchInputButton} onClick={onClick}>
```
Comment (Simon-Laux) [inline src/renderer/components/SearchInput/SearchInputButton.tsx:17]: aria label for accessibility would be nice (not our current focus, but we will need to fix accessibility in the future)

**Review by Simon-Laux — APPROVED** [review]
I like the idea of replacing the item, not sure if that will affect accessibility, but I (as a non-impaired person) like it.

File: images/icons/qr.svg
Comment (Simon-Laux) [reply images/icons/qr.svg:1]: I will make a quick follow-up pr for this, as I already have the editor open

Comment (Simon-Laux) [reply images/icons/qr.svg:1]: https://github.com/deltachat/deltachat-desktop/pull/3666

Comment (adzialocha) [reply images/icons/qr.svg:1]: Nice, thanks! I've also meant to do the refactor in a separate PR.

File: src/renderer/components/SearchInput/index.tsx [hunk 2]
```diff
@@ -35,11 +54,21 @@ export default function SearchInput(props: Props) {
...
+      {hasValue && (
+        <SearchInputButton
+          aria-label='Clear'
+          icon='cross'
+          onClick={handleClear}
+        />
+      )}
+      {!hasValue && (
+        <SearchInputButton
+          aria-label='Scan QR Code'
```
Comment (Simon-Laux) [inline src/renderer/components/SearchInput/index.tsx:66]: we should also translate these labels

**Review by Simon-Laux — APPROVED** [review]

**Pull request merged**